akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ActorContext.scala

```diff
@@ -40,56 +40,78 @@ trait ActorContext[T] {
 
   /**
    * Get the `scaladsl` of this `ActorContext`.
+   *
+   * This method is thread-safe and can be called from other threads than the ordinary
+   * actor message processing thread, such as CompletionStage callbacks.
    */
   def asScala: akka.actor.typed.scaladsl.ActorContext[T]
 
   /**
    * The identity of this Actor, bound to the lifecycle of this Actor instance.
    * An Actor with the same name that lives before or after this instance will
    * have a different [[ActorRef]].
+   *
+   * This method is thread-safe and can be called from other threads than the ordinary
```

Comment:
when porting to scala I'd not say method for `self`, perhaps "field" or "member"

Reply:
but for java it's fine, everything are methods

Reply:
Fixed